src/common/StatusArg.h

```diff
@@ -221,6 +221,12 @@ class StatusVector : public Base
 		return *this;
 	}
 
+	StatusVector& operator<<(const std::string_view& text) noexcept
```

Comment:
It's generally recommended to use `std::string_view` (not const reference) as parameter.